Sources/PackageDescription/Version+StringLiteralConvertible.swift

```diff
@@ -12,7 +12,9 @@
 
 extension Version: ExpressibleByStringLiteral {
     /// Initializes a version struct with the provided string literal.
-    /// - Parameter version: A string literal to use for creating a new version struct.
+    ///
+    /// - Parameters:
+    ///   - version: A string literal to use for creating a new version struct.
```

Comment:
I think the current style where single paramenter uses `- Parameter parameter: description` is better, because its compactness reduces visual clatter around the code. However, it's only a matter of preference.